app/views/categories/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<main>
+  <% if @categories.blank? %>
+    <h3 class="empty">Your categories list is currently empty!</h3>
+  <% else %>
+    <ul class="cards">
+      <% @categories.each do |category| %>
+        <li class="card">
+          <%= link_to user_category_activities_path(@user, category) do %>
+            <div class="card-body d-flex">
+              <div class="image">
+                <% if category.icon.attached? %>
+                  <%= image_tag url_for(category.icon), class: "image" %>
+                <% end %>
+              </div>
+
+              <div class="card-description">
+              <div>
+                <h4><%= category.name %></h4>
+                <p><%= category.created_at %></p>
+              </div>
+              <p>$<%= category.activities.sum { |activity| activity.amount } %></p>
+              </div>
+            </div>
+          <% end %>
+        </li>
+      <% end %>
+    </ul>
+  <% end %>
+</main>
+<div class="add_btn_wrap">
+  <%= link_to "Add new category", new_user_category_path(@user), class: "btn" %>
+</div>
```

Comment:
- [ ] Kindly design the category cards to match the design. You need to format your text and date better. The card should also have some more width to match the design.

- [ ] Kindly design the category page to follow the design. The Add new category button should stick to the bottom of the page and should have the same color as the design with the white inner boundary.

  Your Implementation | Behance Design
  :---:|:---:
  ![image](https://github.com/Kaghenimbale/budget-app/assets/36542986/f0a3322b-e4dd-4386-ac37-053213d6127e) | ![image](https://github.com/Kaghenimbale/budget-app/assets/36542986/88a83df4-cf8c-45ae-b7f5-a0fa9cdb7dd6)